server/py/services/api/api/endpoints/model_endpoints.py

```diff
@@ -381,7 +419,7 @@ async def get_metrics_by_multiple_endpoints(
     project: ProjectAnnotation,
     auth_info: schemas.AuthInfo = Depends(framework.api.deps.authenticate_request),
     type: Literal["results", "metrics", "all"] = "all",
-    endpoint_ids: list[EndpointIDAnnotation] = Query(None, alias="endpoint-id"),
+    endpoint_ids: list[EndpointIDAnnotation] = Query([], alias="endpoint-id"),
```

Comment:
Rationale:
* `endpoint_ids=None` will cause an error to be raised
* `Query([], ...)` in all other relevant places
* I verified that using an empty list as a default is safe in FastAPI (unlike a regular default parameter in python, where it's a no-no)